lib/Backends/Interpreter/Interpreter.cpp

```diff
@@ -365,11 +365,25 @@ bool Interpreter::isOpSupported(const NodeInfo &NI) const {
                   FusedRowwiseQuantizedSparseLengthsWeightedSumNode::
                       ResultIdx) == ElemKind::Float16Ty);
     case ElemKind::UInt8FusedQTy:
-      return (NI.getInElemTy(FusedRowwiseQuantizedSparseLengthsWeightedSumNode::
-                                 WeightsIdx) == ElemKind::FloatTy) &&
-             (NI.getOutElemTy(
-                  FusedRowwiseQuantizedSparseLengthsWeightedSumNode::
-                      ResultIdx) == ElemKind::FloatTy);
+      return ((NI.getInElemTy(
```

Comment:
@SameerAsal  Could you split this huge expression into multiple parts? It is a bit hard to read it.

May be something like:
if (part1) {
  return true;
}
return part2;

or something like that.